HpToolsLauncher/Runners/AlmTestSetsRunner.cs

```diff
@@ -1691,7 +1696,7 @@ private void UpdateCounters(TestRunResults test, TestSuiteRunResults testSuite)
         /// </summary>
         /// <param name="qcTestStatus"></param>
         /// <returns></returns>
-        private TestState GetTsStateFromQcState(string qcTestStatus)
+        private TestState GetTsStateFromQcState(TestExecStatus qcTestStatus)
```

Comment:
Modified the function definition to better handle what is happening inside the function. We need the TestExecStatus instance inside the function body, to determine the correct status of the test.